riscv-runtime/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "powdr-riscv-runtime"
 description = "powdr runtime provider for RISCV programs"
-version = "0.1.0-alpha.2"
+version = "0.2.0-alpha.0"
```

Comment:
Is this an incompatible change that warrants a version bump? I am not sure...

Reply:
the current version in the file is the next to be released, I think it's fine to keep it like that

Reply:
Reverted.